src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -672,13 +670,14 @@ private boolean checkThatEntriesHaveKeys(List<BibEntry> entries) {
     }
 
     private void showConnectionLostErrorMessage() {
-        JOptionPane.showMessageDialog(null,
+        frame.getDialogService().showErrorDialogAndWait(Localization.lang("Connection lost"),
                 Localization.lang("Connection to OpenOffice/LibreOffice has been lost. "
-                        + "Please make sure OpenOffice/LibreOffice is running, and try to reconnect."),
-                Localization.lang("Connection lost"), JOptionPane.ERROR_MESSAGE);
+                + "Please make sure OpenOffice/LibreOffice is running, and try to reconnect."));
+
     }
 
     private void reportUndefinedParagraphFormat(UndefinedParagraphFormatException ex) {
+        //TODO: Javafx does not support html as far as I know
```

Comment:
Thats right, but the dialog does not seem to actually need html (the only tag is for the newline, right?)